mindsdb/migrations/env.py

```diff
@@ -22,9 +22,16 @@
 import os
 
 
-# init database
 
+from mindsdb.utilities.config import Config
 from mindsdb.interfaces.storage import db
+
+# initialize config
+if not 'MINDSDB_CONFIG_PATH' in os.environ:
+    os.environ['MINDSDB_CONFIG_PATH'] = 'absent'
+Config()
+
+# init database
```

Comment:
PEP8: `Inline comments are unnecessary and in fact distracting if they state the obvious` :)

Reply:
changed this piece a little